examples/angular/basic-external-atoms/src/app/app.html

```
@@ -0,0 +1,53 @@
+      @for (row of table.getRowModel().rows; track row.id) {
+        <tr>
+          @for (cell of row.getAllCells(); track cell.id) {
+            <td><ng-container *flexRenderCell="cell; let renderCell">{{ renderCell }}</ng-container></td>
+          }
+        </tr>
+      }
+    </tbody>
+  </table>
+  <div class="spacer-sm"></div>
+  <div class="controls">
+    <button class="demo-button demo-button-sm" (click)="table.setPageIndex(0)" [disabled]="!table.getCanPreviousPage()"><<</button>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)